hummingbot/market/markets_recorder.py

```diff
@@ -233,6 +235,13 @@ def _did_fill_order(self,
         session.add(trade_fill_record)
         self.save_market_states(self._config_file_path, market, no_commit=True)
         session.commit()
+        self.append_to_csv(trade_fill_record)
+
+    def append_to_csv(self, trade: TradeFill):
+        csv_file = CONF_FILE_PATH + "/trades_" + trade.config_file_path + ".csv"
+        df = pd.DataFrame([[trade.config_file_path, trade.strategy, trade.market, trade.symbol, trade.base_asset, trade.quote_asset, trade.timestamp,
+                            trade.order_id, trade.trade_type, trade.order_type, trade.price, trade.amount, trade.trade_fee, trade.exchange_trade_id]])
```

Comment:
Can you add the order age here also, so that everything we show in `history` is also in the exported CSV?

Reply:
Done!